src/main.cpp

```diff
@@ -707,6 +769,8 @@ char const* machinestateEnumToString(const MachineState machineState) {
             return "Manual Flush";
         case kSteam:
             return "Steam";
+        case kWater:
```

Comment:
case kHotWater:
   return "Hot Water";
   
 
 and the order like suggested in the machine state enum